omero/features/analyze/index.html

```diff
@@ -56,41 +62,53 @@ <h4>Scripting service</h4>
                 <a class="button hollow tiny" href="https://www.youtube.com/watch?v=sKRvYVZksdM" target="_blank">View movie</a>
             </div>
             <div class="omero-feature-container tall-content column text-center">
-                <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_matlab.png">
+                <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_python.svg">
+                <h4>Python Blitz API</h4>
```

Comment:
Not introduced by this PR but this heading feels incorrect as it looks like it's referring to the Python part of OMERO.blitz. Is this rather about the `omero.gateway.BlitzGateway` API shipped with `omero-py1?

Reply:
To match the rest, it could be ``Python and OMERO``

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/401/commits/6922f34e5392741713d27a627479bd43891d8f61